<?xml version="1.0"?>
<launch>
  <arg name="image" default="/usb_cam/image_raw" />
  <arg name="model_name" default="midas" />
  <!--Options: "DPT_Large", "DPT_Hybrid", "MiDaS_small"-->
  <arg name="version" default="DPT_Hybrid" />

  <node pkg="pytorch_ros" type="depth_estimation.py" name="depth_estimation" output="screen">
    <remap from="image" to="$(arg image)" />
    <param name="model_name" value="$(arg model_name)" />
    <param name="version" type="str" value="$(arg version)" />
  </node>
</launch>